guides/testing.md

```diff
@@ -108,18 +113,75 @@ end
 
 They return the rescued exception so it can be used for further expectations, for example to verify specific properties of the exception.
 
+## Focusing on a group of specs
+
+A `describe`, `context` or `it` can be marked with `focus: true`, like this:
```

Comment:
```suggestion
`describe`, `context` and `it` blocks can be marked with `focus: true`, like this:
```

Reply:
Note that the original code has this same text as a comment so it should probably be updated there as well.